providers/flagsmith/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/open-feature/go-sdk-contrib/providers/flagsmith
+
+go 1.20
```

Comment:
Are you targeting go version 1.20? Seems your client has support for 1.18 [1], so I assume this was unintentional :) 

[1] - https://github.com/Flagsmith/flagsmith-go-client/blob/main/go.mod#L3

Reply:
Ah, yes, very good point